pkg/apis/pipeline/pod/template.go

```diff
@@ -101,6 +101,13 @@ type Template struct {
 	// +optional
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
 
+	// HostAliases is an optional list of hosts and IPs that will be injected into the pod's hosts
+	// file if specified. This is only valid for non-hostNetwork pods.
+	// +optional
+	// +patchMergeKey=ip
+	// +patchStrategy=merge
+	HostAliases []corev1.HostAlias `json:"hostAliases,omitempty" patchStrategy:"merge" patchMergeKey:"ip" protobuf:"bytes,23,rep,name=hostAliases"`
```

Comment:
I don't think strategic merge patch is supported on CRDs. We also don't have `patchStrategy` and `patchMergeKey` on any other fields in this struct.  Suggest removing.

Reply:
I made the changes and removed them. However, `volumes` field has `patchStrategy ` and `patchMergeKey`, just to say.

Reply:
Ah, you are right, thanks for pointing that out - I missed volumes when i looked!